mnist/mnist_azure.ipynb

```diff
@@ -0,0 +1,968 @@
+{
```

Comment:
<p>Will this work on Azure? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/examples/pull/759/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
It doesn't. That ingress controller doesn't exist. Thanks for catching it.